frontend/src/components/Address.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import { GeoapifyGeocoderAutocomplete, GeoapifyContext } from '@geoapify/react-geocoder-autocomplete'
+import '@geoapify/geocoder-autocomplete/styles/minimal.css'
+
+const Address = () => {
+
+  const geoapifyKey = process.env.REACT_APP_GEOAPIFY_KEY
```

Comment:
Very nice, there are a couple of different ways applying env variables with libraries but this is state of the art